plugin/adsk/scripts/mayaUsdMenu.mel

```diff
@@ -101,15 +101,17 @@ proc initRuntimeCommands() {
             mayaUsdCreateStageFromFileOptions;
     }    
 
-    if (!`runTimeCommand -exists mayaUsdOpenUsdLayerEditor`) {
-        runTimeCommand -default true
-            -label 		"USD Layer Editor"
-            -annotation "Organize and edit USD data in layers"
-            -category   "Menu items.Common.Windows.General Editors"
-            -command 	"mayaUsdLayerEditorWindow mayaUsdLayerEditor"
-            -image		"USD_generic.png"
-            mayaUsdOpenUsdLayerEditor;
-    }    
+    if (`exists mayaUsdLayerEditorWindow`) {
+        if (!`runTimeCommand -exists mayaUsdOpenUsdLayerEditor`) {
+            runTimeCommand -default true
+                -label 		"USD Layer Editor"
+                -annotation "Organize and edit USD data in layers"
+                -category   "Menu items.Common.Windows.General Editors"
+                -command 	"mayaUsdLayerEditorWindow mayaUsdLayerEditor"
+                -image		"USD_generic.png"
+                mayaUsdOpenUsdLayerEditor;
+        }
+    }
```

Comment:
Don't create the runtime command to open the LE if there is no command for the LE window.

Reply:
All of these "exists" checks are for the existence of the command and not the UI right?

Reply:
Yes. One of my other changes is only register the "mayaUsdLayerEditorWindow" command when the Layer Editor was built (i.e. Qt is found). So if we don't have this command we won't create the runtime command to open the LE either.